base_classes/NXaperture.nxdl.xml

```diff
@@ -27,25 +27,39 @@
     name="NXaperture" 
     type="group" extends="NXobject">
 
-    <doc>A beamline aperture.</doc>
+    <doc>A beamline aperture. This group is deprecated, use NXslit instead.</doc>
     <!-- TODO compare with "screens" in SHADOW -->
 
-    <group type="NXgeometry">
+    <group type="NXgeometry" deprecated="Use NXtransformations for positioning the aperture and NXoff_geometry to describe its shape">
         <doc>
-            location and shape of aperture
-            
-            .. TODO: documentation needs improvement, contributions welcome
-            
-                * description of terms is poor and leaves much to interpretation
-                * Describe what is meant by translation _here_ and ...
-                * Similar throughout base classes
-                * Some base classes do this much better
-                * Such as where is the gap written?
+            location and shape of aperture.
         </doc>
     </group>
-    <group name="BLADE_GEOMETRY" type="NXgeometry">
+    <group name="BLADE_GEOMETRY" type="NXgeometry" deprecated="Use NXoff_geometry instead to describe the shape of the aperture">
         <doc>location and shape of each blade</doc>
     </group>
+    <group type="NXoff_geometry" minOccurs="0">
+      <doc>
+        This group contains the geometry description of the aperture and replaces BLADE_GEOMETRY.
+      </doc>
+    </group>
+    <field name="depends_on" type="NX_CHAR">
+      <doc>
+    NeXus positions components by applying a set of translations and rotations 
+    to apply to the component starting from 0, 0, 0. The order of these operations 
+    is critical and form a what NeXus calls a dependency chain. The depends_on 
+    field defines the path to the top most operation of the dependency chain. 
+    Usually these operations are stored in a NXtransformations group. But NeXus 
+    allows them to be stored anywhere.
+      </doc>
+    </field>
+    <group type="NXtransformations" minOccurs="0">
+      <doc>
+    This is the group recommended for holding the chain of translation 
+    and rotation operations necessary to position the component within
+    the instrument. 
```

Comment:
Maybe worth adding something like "The chain may traverse similar groups in other component classes"? The current wording might imply that the full chain for this component must be inside this group.